config/initializers/active_record_extension.rb

```diff
@@ -29,93 +29,26 @@ def assign_or_new(attributes)
   # order: (String) order direction (ASC | DESC)
   # sample: CamaleonCms::Site.first.posts.sort_by_field("untitled-field-attributes", "desc")
   def sort_by_field(key, order = 'ASC')
```

Comment:
Could the module on L23 be mixed into ApplicationRecord or CamaleonRecord instead? It appears to be the only modification to ActiveRecord::Base left in this file.

Reply:
Thanks for great suggestions, @brian-kephart!

Fixed in https://github.com/owen2345/camaleon-cms/pull/1043/commits/051562c4ab62a6e75b9e5d0d090ac0bd160c6c11

Reply:
Love it!